mmv1/third_party/terraform/tests/resource_gke_hub_feature_membership_test.go.erb

```diff
@@ -601,7 +601,18 @@ func TestAccGkeHubFeatureMembership_gkehubFeatureMesh(t *testing.T) {
 				ImportStateVerify: true,
 			},
 			{
-				Config: testAccGkeHubFeatureMembership_gkehubFeatureMeshUpdate(context),
+				Config: testAccGkeHubFeatureMembership_meshUpdateManagement(context),
```

Comment:
This step is currently updating `mesh.management` and also unsetting `mesh.control_plane` - just wanted to double-check that's intentional since there are separate functions for testing those updates separately? Either way is fine from the TF perspective.

Reply:
It is intentional to have this setup, as these cover the "UNSPECIFIED" options too for both the fields.